public/app/percona/shared/core/selectors.ts

```diff
@@ -8,5 +8,6 @@ export const getPerconaUser = (state: StoreState) => state.percona.user;
 export const getKubernetes = (state: StoreState) => state.percona.kubernetes;
 export const getDeleteKubernetes = (state: StoreState) => state.percona.deleteKubernetes;
 export const getAddKubernetes = (state: StoreState) => state.percona.addKubernetes;
+export const getAddDbCluster = (state: StoreState) => state.percona.addDbCluster;
```

Comment:
I realized what the problem was. The name of the `dbcluster` in state is written in the singular and this confused me, I think we should change it to `dbClusters`, and `DBClusterReducer` too (`DBClusterReducer` => `DBClustersReducer`), because` DBClusterReducer` use `fetchDBClustersAction`, wich returns arrays, if you agree with it, can I ask you to change it in this PR?

Reply:
Changed to plural 👍